eyeshade/migrations/0019_add_inserted_at/up.sql

```diff
@@ -0,0 +1,12 @@
+select execute($$
+
+insert into migrations (id, description) values ('0019', 'add_inserted_at');
+
+alter table transactions add column
+inserted_at timestamp with time zone
+not null
+default current_timestamp;
+
+update transactions set inserted_at = created_at;
```

Comment:
I worry about this since I think this will create a lock on every row in the transactions table until the statement completes

Reply:
unsure, but we can take it out